packages/odyssey-storybook/src/contributing/CustomTheme.stories.tsx

```diff
@@ -109,3 +104,43 @@ export const RadioGroupStory: StoryFn = () => {
 };
 
 RadioGroupStory.storyName = "RadioGroup";
+
+export const CustomComponentStory: StoryFn = () => {
```

Comment:
this is just an example to show the customization of mui components. I'll remove this if we don't want this to be in the storybook.